foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -28,7 +28,7 @@ If you are having trouble:
 
 -   Refer to the [workflow](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/git-basics) in the Git Basics Lesson.
 
-#### Tips on When to Commit
+### Tips on When to Commit
```

Comment:
Can you explain why you think this should be a level 3 heading?

To me, it seems like a sub-section of "Setting Up Your Project’s GitHub Repository", especially because it is a brief recap of previously covered material.

Reply:
There are many places were subheadings are used. Here is one from a lesson before this one, though there are many others after this lesson as well:
https://www.theodinproject.com/lessons/foundations-asking-for-help#tips-for-getting-the-best-help-possible